frontend/src/components/AgencyProfile/LocationAndDistributions.jsx

```diff
@@ -0,0 +1,141 @@
+    if (types[1]) {
+      displayedStr = displayedStr + "Meal Program, ";
+    }
+    if (types[2]) {
+      displayedStr = displayedStr + "Homebound Delivery, ";
+    }
+    if (types[3]) {
+      displayedStr = displayedStr + "Large Scale Distribution, ";
+    }
+    if (types[4]) {
+      displayedStr = displayedStr + "Residential Facility, ";
+    }
+    displayedStr = displayedStr.replace(/,\s*$/, "");
+    return displayedStr;
+  }
+
+  /**
+   * Function returns a checkmark if bool is true, otherwise returns a xmark
+   *
+   * @param {*} bool
+   * @returns {*} Img elem
+   */
+  function displayCheckMark(bool) {
+    if (bool) {
+      return <img className="checks" src={checkmark} alt="checkmark"></img>;
+    } else {
+      return <img className="checks" src={xmark} alt="xmark"></img>;
+    }
+  }
+
+  return (
+    <>
+      <div className="agency-category">
+        <img id="edit-icon" src={edit} alt="edit"></img>
+        <h1 className="category-title">LOCATION AND DISTRIBUTION HOURS</h1>
+        <div className="region-container">
+          <p>
+            <strong>Region:</strong>
+            &nbsp;{agency.tableContent.region}
+          </p>
+          <p>
+            <strong>San Diego District:</strong>
+            &nbsp;{agency.sanDiegoDistrict} | &nbsp;
+            <strong>County District:</strong>
+            &nbsp;{agency.countyDistrict} | &nbsp;
+            <strong>State Assembly District:</strong>
+            &nbsp;{agency.stateAssemblyDistrict}
+          </p>
+          <p>
+            <strong>State Senate District:</strong>
+            &nbsp;{agency.stateSenateDistrict} | &nbsp;
+            <strong>Federal Congressional District:</strong>
+            &nbsp;{agency.federalCongressionalDistrict}
+          </p>
+        </div>
+        <div className="address-container">
+          <p>
+            <strong>Additional Site:</strong>
+            &nbsp;
+            {agency.additionalAddresses.map((add, index) => {
+              if (agency.additionalAddresses.length - 1 <= index) {
+                return add + "";
+              } else {
+                return add + ", ";
+              }
+            })}
+          </p>
+          <p>
+            <strong>Billing Address and Zipcode:</strong>
+            &nbsp;{agency.billingAddress}, {agency.billingZipcode}
+          </p>
+          <p>
+            <strong>Distribution Type:</strong>
+            &nbsp;{displayTypes(distributionTypes)}
+          </p>
+        </div>
+        <div className="distributions-container">
+          <div className="distributions-label">
+            <p>
+              <strong>Distribution Days:</strong>
+            </p>
+            <p>
+              <strong>Frequency: </strong>
+              &nbsp;{agency.distributionFrequency}
+            </p>
+          </div>
+          <div className="distribution-days-container">
+            {Object.keys(agency.distributionDays).map((key) => {
+              if (key != "_id") {
```

Comment:
Not sure we should rely on the order of these keys being correct. You could make an array like with `distributionTypes` at the top so you have control over the order:
```javascript
const distributionDays = [
  agency.distributionDays.monday,
  agency.distributionDays.tuesday,
  ...
];
```
Then do a map here:
```javascript
{distributionDays.map((day) => (
  <div className="distribution-day">
    ...
  </div>
))}
```